packages/components/src/theme/stories/index.tsx

```diff
@@ -45,3 +48,54 @@ export const Nested: ComponentStory< typeof Theme > = ( args ) => (
 Nested.args = {
 	accent: 'blue',
 };
+
+/**
+ * The rest of the required colors are generated based on the given accent and background colors.
+ */
+export const ColorScheme: ComponentStory< typeof Theme > = ( {
```

Comment:
Would it be useful for it to work? I was kind of assuming that all the Theme stories are intended to be manipulated directly with controls.

Reply:
Probably not, but it may be confusing. I wonder if we should add a note in the Storybook example saying that changing the theme in the toolbar doesn't work for the specific example, or if there's even a way to disable that toolbar button on demand based on what story is being shown.

Anyway, very minor task and definitely not blocking.

Reply:
Ok. I disabled the toolbar addon for Theme stories, and added a helpful note so it's not confusing: f5f0d63e440b0d8380d2f45166e06435d577d649